net.crossfireatlas.crossfire.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Categories=Game;
+Comment=Roguelike multi-user RPG
+Exec=net.crossfireatlas.crossfire.start.sh
+Icon=net.crossfireatlas.crossfire.ico
```

Comment:
1. .ico isn't a valid icon format. This is not MS-DOS
2. It's the *NAME*. No extension.

Reply:
OK, .ico removed.